docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.            while True:
</pre><p>From personal experience, I always try to avoid using a <code>while</code>  loop because it can potentially run forever and never exit the loop. We should try to replace this with some sort of rational, informed for-loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
I understand. Please let me think about it.

Reply:
Based on your suggestion on decreasing the rate of change linearly, we can now replace it with `for-loop`. I think we should better avoid getting into `min_dist=0`, because it will just return all subsequences as candidates at the end of phase II. Instead, we can define a threshold. For instance, we can say a discord must have **at most** `0.99` correlation  with its NN. This `0.99` can be used to calculate the threshold for `min_dist` (or the dot product value). 

What do you think?

Reply:
<p>From a design stand point, whatever input parameter we choose to allow the user to control things, it needs to be really obvious and intuitive as to what relationship that parameter has with controlling things. I don't think 0.99 correlation is intuitive. I don't know what the answer should be but we should think about this more.</p>

Reply:
I understand. So, for now, I am not going to add this additional parameter. I will make a note.